### PR TITLE
Feature/7087 statistic data local 7 day incidence/7689 local statistics card

### PR DESCRIPTION
**Pull request by 30mar**
## Description
Add the local incidences card
Unit and UI tests are in progress and will be published in another PR soon

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7689

## Screenshots
https://user-images.githubusercontent.com/15270737/123877603-cc5fe080-d93d-11eb-9d40-71336bf60c4a.MP4

### CODE REVIEW

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/AddStatisticsCardView/LocalStatisticsDistrict.swift [hunk 1]
```diff
@@ -0,0 +1,11 @@
+////
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/AddStatisticsCardView/LocalStatisticsDistrict.swift:1]:
```suggestion
//
```

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/Extensions/SAP_Internal_Stats_SevenDayIncidenceData+Formatting.swift [hunk 2]
```diff
@@ -0,0 +1,77 @@
+////
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/Extensions/SAP_Internal_Stats_SevenDayIncidenceData+Formatting.swift:1]:
```suggestion
//
```

**Review by whiskey — APPROVED** [review]
Most of this will be changed later due to the other merge… 😐

**Review by naveeddotio — APPROVED** [review]

**Pull request merged**